client/transaction-pool/graph/src/base_pool.rs

```diff
@@ -83,8 +83,7 @@ pub struct PruneStatus<Hash, Ex> {
 }
 
 /// Immutable transaction
-#[cfg_attr(test, derive(Clone))]
-#[derive(PartialEq, Eq)]
+#[derive(PartialEq, Eq, Clone)]
```

Comment:
The transaction should not be cloneable. It's too heavy to do so frequently. If there is no way to retrieve an `Arc` then please just return `Hash` in the notification stream.